src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenIterators.cs

```diff
@@ -3022,57 +3022,126 @@ public static System.Collections.Generic.IEnumerable<int> Produce()
     }
 }
 """;
-            CompileAndVerify(src, expectedOutput: "42 42").VerifyDiagnostics();
+            var verifier = CompileAndVerify(src, expectedOutput: "42 42").VerifyDiagnostics();
+            verifier.VerifyIL("C.<Produce>d__0.System.IDisposable.Dispose()", """
+{
+  // Code size        1 (0x1)
+  .maxstack  0
+  IL_0000:  ret
+}
+""");
         }
 
         [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/75666")]
+
         public void AddVariableCleanup_StringLocal()
```

Comment:
This shows as an unnecessary empty line between an attribute and the target unit-test.